tests/test_mpi.py

```diff
@@ -2370,9 +2370,9 @@ def test_staggering(self, mode):
         op(time_M=2)
 
         # Expected norms computed "manually" from sequential runs
-        assert np.isclose(norm(ux), 6042.554, rtol=1.e-4)
-        assert np.isclose(norm(uxx), 64632.75, rtol=1.e-4)
-        assert np.isclose(norm(uxy), 59737.77, rtol=1.e-4)
+        assert np.isclose(norm(ux), 7003.098, rtol=1.e-4)
```

Comment:
Why the different norms here? Because something is now interpolated?

Reply:
yes